lib/machine.rb

```diff
@@ -25,4 +25,8 @@ def split_message(message)
     end
   end
 
+  def shift_amount(encrypted_char, end_char)
+    (@alphabet.find_index(encrypted_char) - @alphabet.find_index(end_char)) % 27
+  end
```

Comment:
I moved this method to the parent class because it is shared by KeyFinder and Decryptor and it uses the instance variable alphabet, so it wouldn't work in a module.